src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/Generated/Studio.WebService/Models/AssetItem.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.MachineLearning.Studio.WebService.Models
```

Comment:
Can you reuse "Microsoft.Azure.Management.MachineLearning.WebService.Models" other than dupe the code?